QgisModelBaker/libqgsprojectgen/db_factory/pg_command_config_manager.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+/***************************************************************************
+    begin                :    13/05/19
+    git sha              :    :%H$
+    copyright            :    (C) 2019 by Yesid Polania
+    email                :    yesidpol.3@gmail.com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from .db_command_config_manager import DbCommandConfigManager
+from qgis.PyQt.QtCore import QSettings
+
+
+class PgCommandConfigManager(DbCommandConfigManager):
+
+    _settings_base_path = 'QgisModelBaker/ili2pg/'
```

Comment:
I believe the settings should be in `plugins/QgisModelBaker/ili2pg/`

Reply:
It seems that "Plugins" is being used only for core QGIS plugins. We've always saved Model Baker settings in its own root ('QgisModelBaker'):

![image](https://user-images.githubusercontent.com/652785/58518165-b1757d80-8173-11e9-9cab-90ce90d72180.png)

Reply:
ok, then